paddle/framework/op_registry.cc

```diff
@@ -43,12 +43,13 @@ static VariableNameMap ConvertOpDescVarsToVarNameMap(
   return ret_val;
 }
 
-std::unique_ptr<OperatorBase> OpRegistry::CreateOp(const OpDesc& op_desc) {
+std::unique_ptr<OperatorBase> OpRegistry::CreateOp(const OpDesc& op_desc,
```

Comment:
desc -> program

It is reasonable to add `ProgramDesc` as a parameter of `CreateOp`, as it specifies the context.